src/server/game/Spells/Spell.h

```diff
@@ -234,6 +234,8 @@ struct ChannelTargetData
     SpellDestination spellDst;
 };
 
+static const uint32 SPELL_INTERRUPT_NONPLAYER = 32747;
```

Comment:
don't use global static here maybe?

Reply:
i know it's doesn't feel right, i could hard code the spell directly xD....... listening proposals

Reply:
One possible change:

change SPELL_INTERRUPT_NONPLAYER to a macro with comments explain what is this variable used for.

Reply:
and place this macro in a better place like, hmmm let's say where other spell macros are located.

Reply:
I think it will be enough change to a macro and write some comments if not i will move it to wherever you guys point to